src/logs/ILog.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright 2024 Ceeblue B.V.
+ * This file is part of https://github.com/CeeblueTV/web-utils which is released under GNU Affero General Public License.
+ * See file LICENSE or go to https://spdx.org/licenses/AGPL-3.0-or-later.html for full license details.
+ */
+
+export interface ILog {
+    error(): void;
```

Comment:
In TypeScript, you can define a getter in an interface by specifying a method with a getter syntax. An interface in TypeScript does not directly define properties as getters or setters, but you can define a function that acts as a getter.

Also they are return `void` when implementation returns object